react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+export function AppDrawer() {
+  const [clicked, setClicked] = useState(false);
+
+  function handleClick() {
+    setClicked(!clicked);
+  }
+
+  return (
+    <>
+      <i className={clicked ? 'hidden' : 'fa-solid fa-bars burger' }
+        onClick={handleClick}></i>
+      <div className={clicked ? 'menu' : 'hidden'}>
+        <h3>Menu</h3>
+        <div className='links'>
+          <a onClick={handleClick}>About</a>
```

Comment:
It looks like it all works. Great! Now try to see if you can log a console message indicating which menu item was clicked.